insights/specs/datasources/ls.py

```diff
@@ -52,7 +54,13 @@ def list_with_la_filtered(broker):
 
 @datasource(HostContext)
 def list_with_lan(broker):
-    return _list_items(Specs.ls_lan_dirs)
+    filters = _list_items(Specs.ls_lan_dirs)
+    if 'specs.fstab_mounted' in filters and FSTab in broker:
+        filter_list = filters.split()
```

Comment:
This seems a redundant operation, better update the `_list_items` and let it return a `sorted` list instead. (you may want to update all other `datasources` to `join` the new value returned by `_list_items`